packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewHostApiImpl.java

```diff
@@ -80,7 +79,7 @@ public void setWebContentsDebuggingEnabled(boolean enabled) {
   /** Implementation of {@link WebView} that can be used as a Flutter {@link PlatformView}s. */
   public static class WebViewPlatformView extends WebView implements PlatformView {
     private WebViewClient currentWebViewClient;
-    private WebChromeClientImpl currentWebChromeClient;
+    private WebChromeClientHostApiImpl.SecureWebChromeClient currentWebChromeClient;
```

Comment:
Couldn't the type here just be `WebChromeClient` since it's only being accessed via overridden methods?

Reply:
This could be set to `WebChromeClient`, but then the overridden methods would have to cast this value.